dotcom-rendering/src/web/components/Figure.tsx

```diff
@@ -175,13 +175,19 @@ export const Figure = ({
 			</figure>
 		);
 	}
+
+	let spacefinderComponent: 'rich-link' | 'numbered-list-title' | null = null;
```

Comment:
Might be help to have some comments describing how this will be used?

Reply:
Good idea, I'll add

Reply:
Yeah, that's a fair point, I'll use `undefined` instead